docs/content/concepts/repositories-workspaces/workspaces.mdx

```diff
@@ -1,201 +1,281 @@
 ---
-title: Workspaces | Dagster
-description: Dagster Workspaces are collections of user-defined repositories and information about where they reside.
+title: Workspaces | Dagster Docs
```

Comment:
So I think we can talk about "Workspace files" rather than "workspaces" in this guide. The workspace.yaml file is really the only manifestation left and I think introducing a file rather than a concept makes this feel less heavy.